package.json

```diff
@@ -80,7 +81,7 @@
     "sass-loader": "^8.0.2",
     "semantic-release": "^17.0.2",
     "style-loader": "^1.1.3",
-    "webex": "^1.80.136",
+    "webex": "^1.96.1",
```

Comment:
Can we get this removed from the dev dependencies?

Reply:
It is needed to run locally until we get the documentation on installing peer dependencies added.

Reply:
sure